HighwayPatrol.cpp

```diff
@@ -1,19 +1,25 @@
 #include "HighwayPatrol.h"
 #include <iostream>
 
+#include "Car.h"
+#include "Motorcycle.h"
+#include "SemiTruck.h"
+
 HighwayPatrol::HighwayPatrol() : Vehicle("HighwayPatrol")
 {
 
 }
 
+HighwayPatrol::~HighwayPatrol() {}
+
 void HighwayPatrol::scanHighway(Highway* h)
 {
     std::cout << name << ": scanning highway for speeders" << std::endl;
 
-    for( size_t i = h->vehicles.size(); --i >= 0; )
+    for( size_t i = h->vehicles.size(); i-- > 0; )
     {
         auto* v = h->vehicles[i];
-        if( v->speed > h->speedLimit + 5 )
+        if( v->speed >= h->speedLimit + 5 )
```

Comment:
Don't change code that doesn't need to be changed or shouldn't be changed
- [ ] revert